sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -211,18 +183,17 @@ def receive(self, batch=False, max_batch_size=300, max_wait_time=None):
         self._receive_start_time = self._receive_start_time or time.time()
         deadline = self._receive_start_time + (
             max_wait_time or 0
-        )  # max_wait_time can be None
+        )
         if len(self._message_buffer) < max_batch_size:
             while retried_times <= max_retries:
                 try:
                     if self._open():
-                        self._handler.do_work()  # type: ignore
+                        self._handler.do_work(batch=self._prefetch)  # type: ignore
```

Comment:
Did we decide against doing this?

Reply:
based on our discussion, b/c pyamqp passes in batch to this method in that helper `receive_batch_messages` method, I believe we decided to keep pyamqp as is. I updated uamqp to take kwargs but disregard them, since that's an easier change. (+ In general, I think we want to adjust uamqp to fit pyamqp, rather than the other way around.)